### ISSUE TITLE
darknet_mobilenet pre-trained model

### ISSUE THREAD

**Issue by kingvision** [hjimce/darknet_mobilenet#2]
Hi:
   can you provide a darknet_mobilenet pre-trained model?

thanks

**Comment by TaihuLight** [comment]
https://github.com/zunzhumu/darknet-mobilenet

**Comment by kingvision** [comment]
@TaihuLight Thanks, i use the model, but i detect no any objects. where i should modify the code?
thanks

**Comment by wuxiaomin0110** [comment]
@kingvision   I detect no object, either. Did you find a solution?  Thanks.

**Comment by wuxiaomin0110** [comment]
@hjimce 
Hi:
Is "mobilenet.weights"  the pre-trained model corresponding to cfg/mobilenet_imagenet.cfg?  I use the model, but don't detect anything. How do I adjust the code?  Thank you.

**Comment by hjimce** [comment]
@vmaxxwuxiaomin  mobilenet_imagenet.cfg and pre-trained model mobilenet.weights are  train  for imagenet classification

**Comment by hjimce** [comment]
@vmaxxwuxiaomin no  object detection

**Comment by wuxiaomin0110** [comment]
@hjimce 
OK. Thanks so much. Classification by your model is OK. It's amazing! I wonder if I wanna detect objects, I need to modify the .cfg file and re-train the .weights file, right?

**Comment by hjimce** [comment]
@vmaxxwuxiaomin   yes,you can  modify the mobilenet_imagenet.cfg  like tiny-yolo.cfg ,except the convolution layer

**Comment by wuxiaomin0110** [comment]
@hjimce 
OK. I will try. Thanks so much.

**Comment by 0-LL-0** [comment]
@hjimce Thank you for sharing the code. I want ro know the mobilenet.weights you provide is match with mobilenet_imagenet.cfg? I can run classification command, but can not  get right classification results.

**Comment by hjimce** [comment]
@CBIR-LL  sorry，I forget it。you can  train  it  by  youself  on  imagenet-1k data

**Comment by 0-LL-0** [comment]
@vmaxxwuxiaomin @hjimce Did you try to detect objects using mobilenet network structure ? Is there any improvement in speed ?

**Comment by qxin** [comment]
@vmaxxwuxiaomin hi, how to spell the classification command. thanks ~ (I found it ) it workswell, still thanks. ## ./darknet classify  cfg/mobilenet-1k.cfg mobilenet.weights data/dog.jpg

**Comment by ljh14** [comment]
When I train the model,why does the loss function stop drop down at about 23 ...It is so unbelievable...It makes me so frustrated

**Comment by A-levy** [comment]
@wuxiaomin0110 我最近也在搞这个mobilenet的检测，请问您实现了吗？可以交流一下吗？谢谢！

**Comment by DW1HH** [comment]
@A-levy  你QQ多少

**Comment by aimhabo** [comment]
请问这样的训练情况正常么？感觉loss变成nan有点不对劲
Is this training normal? Feeling loss becomes a bit wrong with nan
![screenshot from 2018-12-26 17-55-50](https://user-images.githubusercontent.com/12708080/50441870-a1910d80-0937-11e9-8f66-300030b8de2e.png)
